packages/cloud-cognitive/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -195,6 +195,13 @@
     white-space: nowrap;
   }
 
+  .#{$block-class}__defaultStringRenderer.#{$block-class}__defaultStringRenderer-multiline {
```

Comment:
I think for BEM, `multiline` should be a modifier so can we rename this to `...__defaultStringRenderer--multiline`?

Reply:
Fixed [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2196/commits/6e2150100777970185d6485f71b4da188a4c0564)